src/content-helper/editor-sidebar/excerpt-suggestions/component-panel.tsx

```diff
@@ -252,22 +276,26 @@ const PostExcerptSuggestions = () => {
 					help={ wordCountString ? wordCountString : null }
 				/>
 			</div>
-			<Button
-				href={ __(
-					'https://wordpress.org/documentation/article/page-post-settings-sidebar/#excerpt',
-					'wp-parsely',
-				) }
-				target="_blank"
-				variant="link"
-			>
-				{ __( 'Learn more about manual excerpts', 'wp-parsely' ) }
-				<Icon
-					icon={ external }
-					size={ 18 }
-					className="parsely-external-link-icon"
-				/>
-			</Button>
-			<div className="wp-parsely-excerpt-generator">
+
+			{ isDocumentSettingPanel && (
+				<Button
+					href={ __(
+						'https://wordpress.org/documentation/article/page-post-settings-sidebar/#excerpt',
+						'wp-parsely',
+					) }
```

Comment:
I think this was made translatable to allow for linking to non-English documentation pages, since this is the official WordPress docs we're linking to.